### PR TITLE
[editor] fix: handle pasted text on multiple blocks including atomic

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/editor/web/src/RichContentEditor/RichContentEditor.jsx [hunk 1]
```diff
@@ -455,7 +427,7 @@ class RichContentEditor extends Component {
         editorState={editorState}
         onChange={this.updateEditorState}
         handleBeforeInput={handleBeforeInput}
-        handlePastedText={this.handlePastedText}
+        handlePastedText={handlePastedText(this.props.handlePastedText, this.updateEditorState)}
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/RichContentEditor.jsx:430]: let's handle both of the params (this.props.handlePastedText, this.updateEditorState) in this file. 
Make the handlePastedText return editorState

File: packages/editor/web/src/RichContentEditor/handlePastedText.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  );
+
+  return changedContentState;
+};
+
+export default (handlePastedText, updateEditorState) => (text, html, editorState) => {
+  if (handlePastedText) {
+    return handlePastedText(text, html, editorState);
+  }
+
```
Comment (shaulgo) [inline packages/editor/web/src/RichContentEditor/handlePastedText.js:47]:
```suggestion
const contentToPaste = html ? draftConvertFromHtml(pastedContentConfig)(HTML) :  ContentState.createFromText(text);
applyPastedContetn(editorState, contentToPaste)
```

File: packages/editor/web/src/RichContentEditor/RichContentEditor.jsx (hunk 1 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/RichContentEditor.jsx]: done.

File: packages/editor/web/src/RichContentEditor/handlePastedText.js (hunk 2 above)
Comment (maimonav) [reply packages/editor/web/src/RichContentEditor/handlePastedText.js]: done.

**Pull request merged**